sklearn/gaussian_process/kernels.py

```diff
@@ -44,6 +44,17 @@ def _check_length_scale(X, length_scale):
     return length_scale
 
 
+def _check_offset(X, c):
```

Comment:
would make sense to unify the two functions. there is no essential difference besides the string in the second ValueError